### PR TITLE
MP4K 3: End of the tournament update

### PR DESCRIPTION
**Pull request by WalterToro**

### CODE REVIEW

**Comment by WalterToro** [comment]
Idk, not many people are available, will update this one tomorrow.

**Comment by cl8n** [comment]
tfw tomorrow turns into the month of July

**Comment by WalterToro** [comment]
I did it mom, I actually finished my homework.

**Comment by MegaApplePi** [comment]
Good to merge now?

**Comment by WalterToro** [comment]
A review can be okay but yeah, it is ready to merge.

File: wiki/Tournaments/MP4K/3/en.md [hunk 1]
```diff
@@ -56,29 +55,332 @@ The minimum guaranteed prizes are:
...
+- Tiebreaker  
+  - **[Quarks (kradness x Camellia) - Intro + Dualive (Evening) \[backpropogation\_edit\]](https://osu.ppy.sh/beatmapsets/996757#mania/2116122)**
+
+### Finals
+
+- FreeMod
+  - [yassu~ - before... (-Rem-) \[mizuki w/ remuther (MP4K3 Edit)\]](https://osu.ppy.sh/beatmapsets/1007040#mania/2107649)
+  - [The Dillinger Escape Plan with Mike Patton - Hollywood Squares (Pope Gadget) \[Noughts\]](https://osu.ppy.sh/beatmapsets/878577#mania/1837470)
+  - [YZYX - .\_Pulse (-Rem-) \[Challenge (240bpm\]](https://osu.ppy.sh/beatmapsets/1007048#mania/2107663)
+  - [guna with Team Kyukkyu - subconsciousness (-Rem-) \[Benzopirene w/ remuther (177bpm\)]](https://osu.ppy.sh/beatmapsets/1006457#mania/2106348)
```
Comment (MegaApplePi) [inline wiki/Tournaments/MP4K/3/en.md]:
```suggestion
  - [guna with Team Kyukkyu - subconsciousness (-Rem-) \[Benzopirene w/ remuther (177bpm\)\]](https://osu.ppy.sh/beatmapsets/1006457#mania/2106348)
```
I'm not exactly sure why but it looks like the other `]` needs to be escaped

Comment (MegaApplePi) [reply wiki/Tournaments/MP4K/3/en.md]: oh lol

Comment (WalterToro) [reply wiki/Tournaments/MP4K/3/en.md]: Yes i noticed this, fixed on old commit

**Pull request merged**